### DISCUSSION TITLE
프로젝트: Kaggle 같은 프로젝트 같이 참가

### DISCUSSION THREAD

**Discussion by bryantson** [Korean-American-IT-Association-KAITA/kaita-project-management#16] [3. 프로젝트 아이디어]
설명: Kaggle (Data Platform) 같은 플랫폼 참가

테크 스택: Kaggle

**Comment by jung-akim** [comment]
https://www.kaggle.com/c/house-prices-advanced-regression-techniques/overview
regression 혹은 time series regression으로 좋을거 같아요

**Comment by jung-akim** [comment]
https://www.kaggle.com/c/contradictory-my-dear-watson/overview
Natural Language Inference (NLI) 저한테 참신한 부분이라 재밌어보이네요 BERT 사용해서 sentiment analysis하듯이 트레이닝 하면 되는 거인지 잘 모르겠지만 파인 튜닝 배우는 데 좋을거 같아요 [여기](https://github.com/yg211/bert_nli) 누가 이미 만든 모델도 있어요
